src/app/kbv/controllers/single-amount-question.js

```diff
@@ -15,15 +15,29 @@ class SingleAmountQuestionController extends BaseController {
         return callback(err, locals);
       }
 
+      if (req.session.question?.info?.months) {
+        locals.threeMonthsAgo = moment()
+          .subtract(req.session.question?.info?.months, "months")
+          .locale(req.lang)
+          .format("DD MMMM YYYY");
```

Comment:
We could abstract this part to a presenter, then we can test it in isolation (including test for welsh) and stub it in the controller tests?